File: S03-conceptos-basicos/CL11-Context/app/src/main/java/com/udemyandroid/ciclovidafragment/MainActivity.java
package com.udemyandroid.ciclovidafragment;

import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements BlankFragment.OnFragmentInteractionListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Toast.makeText(this, "Mensaje desde Activity", Toast.LENGTH_SHORT).show();

    }

    @Override
    public void onFragmentInteraction(Uri uri) {

    }
}

File: S03-conceptos-basicos/CL08RadioButtons/app/src/main/java/com/androidavanzado/cl08radiobuttons/MainActivity.kt
package com.androidavanzado.cl08radiobuttons

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.RadioButton
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sexoSeleccionado(view: View) {
        val radioButtonSeleccionado = view as RadioButton

        when(view.id) {
            R.id.radioButtonMujer -> {
                Toast.makeText(this, "Se ha seleccionado Mujer", Toast.LENGTH_SHORT).show()
            }
            R.id.radioButtonHombre -> {
                Toast.makeText(this, "Se ha seleccionado Hombre", Toast.LENGTH_SHORT).show()
            }
            R.id.radioButtonIndefinido -> {
                Toast.makeText(this, "Se ha seleccionado Indefinido", Toast.LENGTH_SHORT).show()
            }
        }
    }
}
